dotnet3.5/S3/CopyObject/CopyObject.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+// snippet-start:[S3.dotnet35.CopyObject]
+using Amazon;
+using Amazon.S3;
+using Amazon.S3.Model;
+using System;
+using System.Threading.Tasks;
+
+namespace CopyObject
+{
+    public class CopyObject
+    {
+        // Specify your bucket region (an example region is shown).
+        private static readonly RegionEndpoint BUCKET_REGION = RegionEndpoint.USWest2;
+        private static IAmazonS3 _s3Client;
+
+        // Specify the names for the Simple Storage Service objects
```

Comment:
The reason for these constants seem obvious to me, which would make this comment unnecessary. Also, "Amazon Simple Storage Service (Amazon S3)" on first mention in each code file, "Amazon S3" on second mention, and "S3" on tertiary and subsequent mentions.

Reply:
Changed this comment to a reminder to be sure and change these names to objects that the user owns.